src/modules/mavlink/mavlink_main.h

```diff
@@ -166,18 +166,18 @@ class Mavlink : public ModuleParams
 	/**
 	 * Get the MAVLink system id.
 	 *
-	 * @return		The system ID of this vehicle
+	 * @return The system ID of this vehicle
 	 */
 	int			get_system_id() const { return mavlink_system.sysid; }
 
 	/**
 	 * Get the MAVLink component id.
 	 *
-	 * @return		The component ID of this vehicle
+	 * @return The component ID of this vehicle
 	 */
 	int			get_component_id() const { return mavlink_system.compid; }
 
-	const char *_device_name;
+	const char *_device_name{DEFAULT_DEVICE_NAME};
```

Comment:
Isn't this still set in the constructor?

Reply:
I left the inits in the constructor in this PR to make the diff a little simpler. I relalize that it's repetitively redundant at the moment but I was aiming to keep the diff as easy as possible to verify the values in this PR.  I can deprecate as I migrate, give me a minute and I'll have that pushed up.

Reply:
The vars that are initialized at declaration in this PR are no longer in the constructor list.  Thanks for your review!